lib/osf-components/addon/components/addons-service/addon-account-setup/component.ts

```diff
@@ -14,17 +14,19 @@ import { AddonCredentialFields } from 'ember-osf-web/models/authorized-account';
 import { CredentialsFormat } from 'ember-osf-web/models/external-service';
 import ExternalStorageServiceModel from 'ember-osf-web/models/external-storage-service';
 import { AllAuthorizedAccountTypes } from 'ember-osf-web/packages/addons-service/provider';
+import captureException, { getApiErrorMessage } from 'ember-osf-web/utils/capture-exception';
 
 // TODO: Get this from GravyValet??
 const repoOptionsObject: Record<string, string[]> = {
     dataverse: [
-        'dataverse.harvard.edu',
-        'dataverse.lib.virginia.edu',
+        'https://dataverse.harvard.edu',
+        'https://dataverse.lib.virginia.edu',
     ],
     gitlab: [
         'https://gitlab.com',
     ],
 };
+
```

Comment:
Hmmm. What about users who want to specify their own dataverse or gitlab instances?

Reply:
at the moment, the compromise is the `configurable_api_root` attr -- if the external-service has `configurable_api_root: true` (like we'll expect for gitlab, in this backcompat-preserving move), then the user's url can go in the authorized-account's `api_base_url`

Reply:
That seems fine to me, but two things:

1. We'll want to check with Product to make sure they are okay with the multiple services per host thing.
2. This will complicate the credentials migration.

Reply:
I agree that this should be backend data, but I'm a little reluctant about having 3 different dataverse implementations. Could it be possible to have some new `AddonOperationInvocation` operation type that would support getting possible base url options? Then we could just call this new operation for any external-service that has `configurableApiRoot`?

Reply:
if you must, can get the full list of `external-storage-services` (won't be crazy long), group them by `addon_imp`, and gather the `api_base_url` from each, but i think actually showing a distinct `external-service` for services on distinct domains (note they do share the same imp(lamentation)) is a better way to go -- is a user more likely to recognize the name of their institution (as shown in the display name of the `external-service`) or the name of some data-verse software their library uses?

recommend abandoning the old grouping of addons by hard-coded name (assuming exactly one `'dataverse'` addon), it's one of the problems that led us to rewrite

Reply:
(am good with all these possible changes going in another pull)